### DISCUSSION TITLE
Cydia Package URLs in depictions should open the depiction

### DISCUSSION THREAD

**Discussion by absidue** [wstyres/Zebra#752] [Ideas]
**Is your feature request related to a problem? Please describe.**
When tapping Cydia Package URLs in package depictions currently nothing happens.

**Describe the solution you'd like**
I would like Zebra to open the depiction of the Cydia Package URL I tapped on.

**Describe alternatives you've considered**
- Leaving it as it is
- Passing the URLs on to Cydia

**Additional context**
- By Cydia URLs I mean the following: `cydia://package/me.absidue.zebraautocomplete`
- A tweak with a few of the URLs in it's depiction is BTStack from the BigBoss repo, namely:
    - BTStack GPS => `cydia://package/ch.ringwald.gps`
    - Blutrol => `cydia://package/ch.ringwald.blutrol`
    - BTStack GPS => `cydia://package/ch.weblooks.webe-plus`

**Comment by PoomSmart** [comment]
Doesn’t this mean Zebra will steal URL events at all cost over Cydia?

**Comment by absidue** [comment]
At the moment the URLs don't do anything.

**Comment by absidue** [comment]
The other option would be to pass all Cydia URLs on to Cydia
